### ISSUE TITLE
Move dependencies

### ISSUE THREAD

**Issue by Semigradsky** [lepture/editor#85]
Move dependencies from vendor folder to package.json

**Comment by lepture** [comment]
Yes, there is a branch https://github.com/lepture/editor/tree/bower

But I don't have much time
